lib/config/guiconfiguration.go

```diff
@@ -9,12 +9,14 @@ package config
 import (
 	"net/url"
 	"os"
+	"strconv"
 	"strings"
 )
 
 type GUIConfiguration struct {
 	Enabled                   bool     `xml:"enabled,attr" json:"enabled" default:"true"`
 	RawAddress                string   `xml:"address" json:"address" default:"127.0.0.1:8384"`
+	UnixPermissions           string   `xml:"unixPerms,omitempty" json:"unixPerms"`
```

Comment:
Please keep the config and json and code variable names the same, preferably verbose.

Reply:
I'd even go all the way to `unixSocketPermissions` to avoid any confusion about where these unix permissions might apply.

Reply:
Done.